c++/src/kj/mutex.h

```diff
@@ -285,6 +289,8 @@ class Once {
     // Fast path check to see if runOnce() would simply return immediately.
 #if KJ_USE_FUTEX
     return __atomic_load_n(&futex, __ATOMIC_ACQUIRE) == INITIALIZED;
+#elif defined(KJ_NO_THREADS)
+    return true;
```

Comment:
This is not correct. When threads are disabled, we still need to run the initializer, and make sure it is run just once. This line appears to result in the initializer never running at all?

Reply:
Valid, I'll fix